src/ui/features/onboarding/Success/WebAppMessageHandler.tsx

```diff
@@ -0,0 +1,82 @@
+import React, { useEffect, useRef } from 'react';
+import { invariant } from 'src/shared/invariant';
+import { isObj } from 'src/shared/isObj';
+
+const ZERION_WEB_APP_URL = new URL(
+  process.env.NODE_ENV === 'development'
+    ? 'http://localhost:3000'
```

Comment:
Looks like this should be a var inside .env file

Reply:
Agree, or just use hardcoded `'https://app.zerion.io'`, and localhost link shouldn't be commited
Whichever's easier

Reply:
I don't think we need `https://localhost:3000` anymore (since it is already merged) ✅ Updated